variables.tf

```diff
@@ -36,7 +36,7 @@ variable "dry_run" {
 variable "lambda" {
   description = "Object of attributes for the lambda function configuration"
   type = object({
-    runtime = optional(string, "python3.9")
+    runtime = optional(string, "python3.8")
```

Comment:
I have changed python ver from 3.8 to 3.9 because of the following error:
<img width="826" alt="Screen Shot 2023-06-12 at 8 11 58 AM" src="https://github.com/plus3it/terraform-aws-codecommit-pr-reminders/assets/123498689/967e974f-1344-4a58-8846-6b5e1d7061f7">
@lorengordon

Reply:
> This is a good start! We also need to update the `lambda` variable and pass through all the values that we use in the other two projects, linked in the task.
> 
> ```
>   artifacts_dir            = var.lambda.artifacts_dir
>   build_in_docker          = var.lambda.build_in_docker
>   create_package           = var.lambda.create_package
>   ephemeral_storage_size   = var.lambda.ephemeral_storage_size
>   ignore_source_code_hash  = var.lambda.ignore_source_code_hash
>   local_existing_package   = var.lambda.local_existing_package
>   memory_size              = var.lambda.memory_size
>   recreate_missing_package = var.lambda.recreate_missing_package
>   runtime                  = var.lambda.runtime
>   s3_bucket                = var.lambda.s3_bucket
>   s3_existing_package      = var.lambda.s3_existing_package
>   s3_prefix                = var.lambda.s3_prefix
>   store_on_s3              = var.lambda.store_on_s3
>   timeout                  = var.lambda.timeout
> 
>   source_path = [
>     {
>       path             = "${path.module}/src"
>       pip_requirements = true
>       patterns         = ["!\\.terragrunt-source-manifest"]
>     }
>   ]
> ```
> 
>  ```
> variable "lambda" {
>   description = "Object of optional attributes passed on to the lambda module"
>   type = object({
>     artifacts_dir            = optional(string, "builds")
>     build_in_docker          = optional(bool, false)
>     create_package           = optional(bool, true)
>     ephemeral_storage_size   = optional(number)
>     ignore_source_code_hash  = optional(bool, true)
>     local_existing_package   = optional(string)
>     memory_size              = optional(number, 128)
>     recreate_missing_package = optional(bool, false)
>     runtime                  = optional(string, "python3.9")
>     s3_bucket                = optional(string)
>     s3_existing_package      = optional(map(string))
>     s3_prefix                = optional(string)
>     store_on_s3              = optional(bool, false)
>     timeout                  = optional(number, 300)
>   })
>   default = {}
> }
> ```

In which repo should the lambda variables be updated ?